f8a_worker/workers/init_package_flow.py

```diff
@@ -115,36 +38,21 @@ def execute(self, arguments):
         except NoResultFound:
             raise FatalTaskError('Unknown ecosystem: %r' % arguments['ecosystem'])
         package = Package.get_or_create(db, ecosystem_id=ecosystem.id, name=arguments['name'])
-        url = self.get_upstream_url(arguments)
-        upstream = self.get_upstream_entry(package, url)
-        if upstream is None:
-            upstream = self.add_or_update_upstream(package, url)
-        arguments['url'] = upstream.url
```

Comment:
I think not having the URL in arguments could cause problems down the road. For example in GitHub task: https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/08f5af72f7f24105541659f103e0e271c6e2d3d2/f8a_worker/workers/githuber.py#L82